docs/pages/versions/unversioned/sdk/speech.md

```diff
@@ -85,6 +102,12 @@ Speak out loud the `text` given `options`. Calling this when another text is bei
   - **onStopped (_function_)** -- A callback that is invoked when speaking is stopped by calling `Speech.stop()`.
   - **onError (_function_)** -- (Android only). A callback that is invoked when an error occurred while speaking.
 
+#### Error Codes
+
+- [`ERR_SPEECH_INPUT_LENGTH`](#err_speech_input_length)
+
+---
```

Comment:
```suggestion
```

Reply:
I think this separator increases readability...

Reply:
For me, it's looking weird, cause other methods aren't separate. It's looking more like the end of the "methods section".